crates/ironrdp-displaycontrol/src/client.rs

```diff
@@ -22,29 +59,12 @@ impl DvcProcessor for DisplayControlClient {
     }
 
     fn process(&mut self, _channel_id: u32, payload: &[u8]) -> PduResult<Vec<DvcMessage>> {
-        // TODO: We can parse the payload here for completeness sake,
-        // in practice we don't need to do anything with the payload.
-        debug!("Got Display PDU of length: {}", payload.len());
-        Ok(Vec::new())
+        let caps = DisplayControlCapabilities::decode(&mut ReadCursor::new(payload))?;
+        debug!("received {:?}", caps);
```

Comment:
```
debug!("Received display control capabilities: {:?}", caps)
```

Reply:
I'm of two minds of this. In the logs, this current version looks like

```
received DisplayControlCapabilities { ... }
```

ergo in

```
Received display control capabilities: DisplayControlCapabilities { ... }
```

the "display control capabilities" is redundant. OTOH "Received display control capabilities:" makes it easier to locate from the logs where in the code we are, because it's (more or less) unique. 

However I think that ultimately we should configure the logger formatting settings to give us that, in order to avoid the messiness/redundancy of relying on the uniqueness of the debug message. In which case keeping it as is will eventually get us annoyed enough to actually fix that.

Reply:
According to our [style guide](https://github.com/Devolutions/IronRDP/blob/master/STYLE.md#logging) log message should start from the capital message.

I agree without you on this one, my suggestion adds redundancy, let it be just `Received DisplayControlCapabilities { ... }` (starting from capital later)